### PR TITLE
Setup navbar and homepage hero title

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Uncomplete hero because this PR is already very long. Also adds a few things:
* Setup root imports
* Setup column layout
* Section component
* Background component

### CODE REVIEW

File: src/components/Navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Link from "gatsby-link";
+import Section from "root/components/Section";
+import Button from "root/components/Button";
+import brandLogo from "root/assets/images/brand.svg";
+import Typography from "../Typography";
+import capitalize from "../../utils/capitalize";
```
Comment (taniaprocha) [inline src/components/Navbar/index.js]: You can use capitalize from lodash #ilovelodash

File: src/components/Section/index.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import "../../styles/breakpoints.css";
+@import "../../styles/layout.css";
+
+.root {
+  @include Breakpoint-desktopOnly {
+    width: $ten-columns;
```
Comment (filipebfernandes) [inline src/components/Section/index.css]: 8 columns

File: src/components/Navbar/index.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  @include Breakpoint-desktopOnly {
+    width: $ten-columns;
+
+    margin: 0 auto;
+  }
+}
+
+.brand {
+  color: white !important;
```
Comment (taniaprocha) [inline src/components/Navbar/index.css]: You don't need `!important`

File: src/components/Section/index.css (hunk 2 above)
Comment (jfranciscosousa) [reply src/components/Section/index.css]: Done thanks

**Pull request merged**